src/plugins/pcr/src/InSilicoPcrWorker.cpp

```diff
@@ -192,6 +192,7 @@ Task* InSilicoPcrWorker::createPrepareTask(U2OpStatus& os) const {
     if (task == nullptr) {
         os.setError(tr("Can not read the primers file: ") + primersUrl);
     }
+    task->moveDocumentToMainThread = true;
```

Comment:
Not really related to the crash, but resolves [this](https://ugene.dev/tracker/browse/UGENE-7785?focusedCommentId=31988&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-31988) unpredictable behaviour

Reply:
ok